cargo-shuttle/src/lib.rs

```diff
@@ -623,6 +630,11 @@ impl Shuttle {
             addr
         );
 
+        println!(
+            "Run `cargo shuttle project start` to create a project environment on Shuttle.\n\
+                  Run `cargo shuttle deploy` to deploy your Shuttle service."
+        );
```

Comment:
I wouldn't add this here or for anything related to the local run. The local run is blocking until the program is stopped (through signals), so it doesn't make too much sense in my opinion to print the next commands at the end of the command after it's stopped by a signal.